swaglyrics/cli.py

```diff
@@ -64,24 +64,27 @@ def get_lyrics(song, artist):
 	:param artist: song artist
 	:return: song lyrics or None if lyrics unavailable
 	"""
-	url_data = stripper(song, artist)  # generate url path using stripper()
-	url = f'https://genius.com/{url_data}-lyrics'  # format the url with the url path
-	try:
-		page = requests.get(url)
-		page.raise_for_status()
-	except requests.exceptions.HTTPError:
-		url_data = requests.get(f'{backend_url}/stripper', data={
-			'song': song,
-			'artist': artist}).text
-		if not url_data:
-			return None
-		url = 'https://genius.com/{}-lyrics'.format(url_data)
-		page = requests.get(url)
+	url_data = stripper(song, artist)
+	if len(url_data) > 0:  # generate url path using stripper()
```

Comment:
This wouldn't work, even if you pass song and artist with empty name, stripper would not be empty. I suggest you do a test run and then see how `stripper()` behaves and how to refactor get_lyrics accordingly